css/properties/shape-outside.json

```diff
@@ -172,6 +172,57 @@
             }
           }
         },
+        "path": {
+          "__compat": {
+            "description": "<code>path()</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/path",
+            "spec_url": "https://w3c.github.io/csswg-drafts/css-shapes/#funcdef-basic-shape-path",
+            "support": {
+              "chrome": {
+                "version_added": "46",
+                "partial_implementation": true,
+                "notes": "Only supported on the <code>offset-path</code> property."
```

Comment:
So, this is confusing. 

> Only supported on the <code>offset-path</code> property.

This sentence makes sense if you are on the [`basic-shape`](https://developer.mozilla.org/en-US/docs/Web/CSS/basic-shape#browser_compatibility) page. But this BCD is for a specific property, [`shape-outside`](https://developer.mozilla.org/en-US/docs/Web/CSS/shape-outside#browser_compatibility), and so I guess the `path()` support should be `false`?